src/commands/create_folder.ts

```diff
@@ -20,19 +21,20 @@ new CLICommand({
 		DryRunParameter,
 		...DrivePrivacyParameters
 	],
-	async action(options) {
+	action: new CLIAction(async function action(options) {
 		const parameters = new ParametersHelper(options);
 		const wallet: Wallet = await parameters.getRequiredWallet();
+		const dryRun = !!parameters.getParameterValue(DryRunParameter);
```

Comment:
Any reason we didn't extend the generic parameter pattern to this parameter getter? 

```suggestion
              const dryRun = parameters.getParameterValue<boolean>(DryRunParameter);
```

Reply:
Well there's something else going on here beyond type mapping which is that we're also choosing a default value in the absence of it existing. e.g. `!!undefined` is `false`. I do think we could improve getParameterValue to do type mapping, but it should also provide for supplying a default value in the absence of a set option. I wanted to do it in this PR but it's too code change for something that won't make it into core right now.